caffe2/opt/onnxifi_transformer.cc

```diff
@@ -89,7 +90,7 @@ std::vector<::ONNX_NAMESPACE::ValueInfoProto> ConvertToValueInfo(
     } else {
       auto* tensor_type = value_info.mutable_type()->mutable_tensor_type();
       tensor_type->set_elem_type(
-          ::ONNX_NAMESPACE::TensorProto_DataType::TensorProto_DataType_FLOAT);
+          onnx::Caffe2TypeToOnnxType(it->second.data_type()));
```

Comment:
Why we are using "onnx" here?

Reply:
caffe2::onnx is a different namespace from ::ONNX_NAMESPACE